### PR TITLE
Update Makefile: Specify vc version for  the build replayability

### PR DESCRIPTION
**Pull request by zaoqi**

### CODE REVIEW

**Comment by aguspiza** [comment]
Why put v.c outside of repo?

File: Makefile [hunk 1]
```diff
@@ -6,8 +6,8 @@ v: v.c
 	./v -o v compiler
 	rm v.c
 
-v.c:
-	curl -Os https://raw.githubusercontent.com/vlang/vc/master/v.c
+v.c: Makefile
```
Comment (Compro-Prasad) [inline Makefile]: Why this?

Comment (zaoqi) [reply Makefile]: Update v.c when the v.c URL in the Makefile is updated

Comment (Compro-Prasad) [reply Makefile]: I get it now. Thanks for the explanation.

Comment (zaoqi) [reply Makefile]: But this is the simplest way.

**Comment by medvednikov** [comment]
I don't want v.c to pollute this repository's history.

Makefile should always have the latest v.c.

**Comment by zaoqi** [comment]
@medvednikov So the V language standard isn't going to change?

**Pull request closed without merging**